cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -84,7 +87,7 @@ def test_delete_static_custom_attributes(provider):
         * The attributes was successfully deleted
         (you should not see a custom attributes table)
     """
-
+    provider.add_custom_attributes(*ATTRIBUTES_DATASET)
     provider.delete_custom_attributes(*ATTRIBUTES_DATASET)
     view = refresh_and_navigate(provider, 'Details')
     if view.entities.custom_attributes.is_displayed:
```

Comment:
this is more lax than docstring — it allows a table that contains other attributes.
since you create and delete them all, I think you can require table to be absent.

[hi :wave:, sorry for drive-by review, reading code in attempt to understand question in email :-)]